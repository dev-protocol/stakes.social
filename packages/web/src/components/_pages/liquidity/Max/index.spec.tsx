import React from 'react'
import { render } from '@testing-library/react'
import { Max } from '.'
import 'src/__mocks__/window/matchMedia.mock'

jest.mock('src/fixtures/_pages/liquidity/geyser/hooks.ts')
jest.mock('src/fixtures/_pages/liquidity/uniswap-pool/hooks.ts')

describe(`${Max.name}`, () => {
  test('Snapshot', () => {
    const component = render(<Max onClick={() => {}}></Max>)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
