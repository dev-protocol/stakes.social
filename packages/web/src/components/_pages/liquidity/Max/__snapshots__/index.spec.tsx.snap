// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Max Snapshot 1`] = `
<body>
  <div>
    <button
      class="ant-btn ant-btn-primary ant-btn-sm sc-bdvvtL cDvhaI"
      type="button"
    >
      <span>
        Max
      </span>
    </button>
  </div>
</body>
`;
